app/components/content/quote_component.html.erb

```diff
@@ -0,0 +1,25 @@
+<blockquote class="<%= quote_class %>">
+  <p><%= text_sans_last_word %><span><%= text_last_word %></span></p>
+  <% if show_footer? %>
+    <footer class="<%= footer_class %>">
```

Comment:
Arguably easier to use Rails' tag helper here, something like. It also takes an array meaning you don't need to `join(" ")` in the method.

```suggestion
    <%= tag.footer(class: footer_class) do %>
```